Dockerfile

```diff
@@ -1,32 +1,62 @@
 #
```

Comment:
@erikdw yeah, I also came to the same questions myself. i think since everything is so configurable because we are coupling two systems, MESOS and STORM, it will be hard to have a base image. So the nature of our build is with onbuilds in the end using arguments passed at build time.